docs/player-support/snapcast.md

```diff
@@ -34,13 +38,13 @@ Snapcast packages are available for several Linux distributions:
 - [Archlinux](#archlinux)
 - [Void Linux](#void-linux)
 
-### Debian
+<h3>Debian</h3>
```

Comment:
These should be the same, but I'd rather stick with the md versions for consistency

Reply:
Dont delberately so they dont get indexed as per here https://github.com/squidfunk/mkdocs-material/issues/313#issuecomment-299698639

Reply:
Fair enough, then that should be the way to go I guess